x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings.tsx

```diff
@@ -40,7 +42,7 @@ export const ResultSettings: React.FC<Props> = ({ engineBreadcrumb }) => {
           <ResultSettingsTable />
         </EuiFlexItem>
         <EuiFlexItem grow={3}>
-          <div>TODO</div>
+          <div>{!dataLoading && <QueryPerformance />}</div>
```

Comment:
Is this extra div wrapper necessary just to check? No worries if it is, AFAIK there's some oddities around EuiBadge inside a flex item

Reply:
Nope, I don't think we need this. I'll remove it.